File: 4.Programming course/C language/gpio.c
#include<stdio.h>
#include<wiringPi.h>

#define GPIO0 0 
int main()
{
	printf("This is an example of controlling the high and low levels of GPIO pin output\n");
	wiringPiSetup();
	pinMode(GPIO0,OUTPUT);
	//printf("Set GPIO0 : H\n");
    //digitalWrite(GPIO0,HIGH); 
	printf("Set GPIO0 : L\n");
	digitalWrite(GPIO0,LOW);
	while(1)
	{

	}
	return 0;
}

File: 4.Programming course/Python language/gpio.py
#/usr/bin/env python3
#coding=utf-8

import wiringpi

GPIO_Pin = 0
OUTPUT = 1
HIGH = 1
LOW = 0

wiringpi.wiringPiSetup()
wiringpi.pinMode(GPIO_Pin,OUTPUT)

while 1:
    print ('Set GPIO_Pin => H')
    wiringpi.digitalWrite(GPIO_Pin,HIGH)
    wiringpi.delay(500)
    print ('Set GPIO_Pin => L')
    wiringpi.digitalWrite(GPIO_Pin,LOW)
    wiringpi.delay(500)